Dockerfile

```diff
@@ -7,10 +7,13 @@ EXPOSE 8080
 # Refer to Maven build -> finalName
 ARG JAR_FILE
 
+COPY . /opt/app
+
 # cd /opt/app
 WORKDIR /opt/app
 
-COPY ${JAR_FILE} app.jar
+RUN ./mvnw clean install
 
+COPY ${JAR_FILE} app.jar
```

Comment:
Just so I understand, we are compiling the Java application using `./mvnw clean install`, and afterwards we are still copying a compiled binary from the host?

I could be misunderstanding, but I'm thinking line 17 shouldn't be here. Have you tried running this on a fresh `git clone`?

Reply:
yup we used a host sources for compiling a java jar . I'll create a new PR with changing to get sources from git clone